lib/url-pattern/url-match-pattern.js

```diff
@@ -64,19 +77,26 @@ _.assign(UrlMatchPattern.prototype, /** @lends UrlMatchPattern.prototype */ {
      */
     createMatchPattern: function () {
         var matchPattern = this.pattern,
+            match;
+
+        // Check the match pattern of sanity and split it into protocol, host and path
 
-            // Check the match pattern of sanity and split it into protocol, host and path
+        if (this._allowProtocols) {
```

Comment:
We should follow a consistent style. I see above that you've used a lot of inlining of conditionals. Why not do the same here?